### ISSUE TITLE
"Back to search results" link on mobile is too small

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1364]
## Description

"Back to search results" link should be at least 40px on mobile to be tappable.

## Reproduction

1. Search for something on search-staging.openverse.engineering using mobile screen.
2. Open one of the results. 
3. You should see "Back to search results" link that only has 28px-high tappable area.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by rutterjt** [comment]
Hey there! I see from your readme that you don't require permission ahead of time to work on issues, but I just wanted to let you know that I can tackle this one.
